src/makeRobotAccountant.js

```diff
@@ -7,6 +7,19 @@
 
 function makeRobotAccountant() {
   // write code here
+  let count = 0;
```

Comment:
Count - verb, counter - noun
```suggestion
  let counter = 0;
```